dll/shellext/cryptext/lang/en-US.rc

```diff
@@ -0,0 +1,14 @@
+/*
+ * PROJECT:     ReactOS CryptExt Shell Extension
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
+ * PURPOSE:     English (United States) resource translation
+ * COPYRIGHT:   Copyright 2019 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+STRINGTABLE
+BEGIN
+    IDS_INVALIDFILE "This is not a valid certificate file"
```

Comment:
Sentence ending with a dot?

Reply:
Why?

Reply:
That's the string that is displayed in the message box.
You sound similar to as if I was asking to put a capital letter at the beginning of the sentence, and you ask "why?".

Reply:
Or better an exclamation mark